TTIP-UNQ-Team10/Mapping-Platform-Backend
# Necessity model

#### This MR contains  
- Modifying of the migration json data  
- Added plugin for cors managing  

#### How to test it  
- Drop the database and then create it again  
- Run `npm install`  
- Run `sudo docker-compose up`  
- Up the server running `npm start`  
- Run the migrations with `./node_modules/.bin/ts-node ./node_modules/typeorm/cli.js migration:run`  
- From the postman client or the frontend client, fetch the necessities with `http://localhost:3001/necessities` and you should see that the necessities coordinates values are an array, and the `Access-Control-Allow-Origin` header is `*`
